src/components/display/fewlines/Languages/Locale.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import styled from "styled-components";
+
+import { H1 } from "../H1/H1";
+import { MagnifyingGlass } from "../Icons/MagnifyingGlas/MagnifyingGlass";
+import { Input } from "../Input/Input";
+import { RadioButton } from "../RadioButton/RadioButton";
+import { Separator } from "../Separator/Separator";
+import { deviceBreakpoints } from "@src/design-system/theme/lightTheme";
+
+export const Locale: React.FC = () => {
+  return (
+    <>
+      <Container>
+        <H1>Switch Language</H1>
+        <div className="search-input">
+          <Input />
+          <div className="magnifying-glass">
+            <MagnifyingGlass />
+          </div>
+        </div>
+      </Container>
+      <LanguagesList>
+        <ListItem>
+          <Value>English</Value>
+          <RadioButton name="locale" checked={true} />
+        </ListItem>
+        <Separator />
+      </LanguagesList>
```

Comment:
❓ Why did you choose to put the language list outside the container?

Reply:
On our wireframe, the `<H1 />` and the `<Input />` are contained in a wrapper with a maximum `width` whereas the list stretches to the entire `width` of the page

![image](https://user-images.githubusercontent.com/33835824/96719952-6bddd400-13aa-11eb-8f61-75982c7148f2.png)